modules/app/stores/uiFilters.ts

```diff
@@ -22,8 +22,6 @@ type Store = {
   resetPollFilters: () => void;
   executiveSortBy: 'Date Posted' | 'MKR Amount';
   setExecutiveSortBy: (method: 'Date Posted' | 'MKR Amount') => void;
-  commentSortBy: 'Latest' | 'Oldest' | 'MKR Amount';
```

Comment:
removed sorting comments from this store, is too generic and sorting comments can be done at a component level